addons/repair/models/repair.py

```diff
@@ -196,6 +196,15 @@ def _default_picking_type_id(self):
     reserve_visible = fields.Boolean(
         'Allowed to Reserve Production', compute='_compute_unreserve_visible',
         help='Technical field to check when we can reserve quantities')
+    picking_type_visible = fields.Boolean(compute='_compute_picking_type_visible')
+
+    def _compute_picking_type_visible(self):
+        for ro in self:
+            picking_count = self.env['stock.picking.type'].search_count([
+                ('code', '=', 'repair_operation'),
+                ('company_id', '=', ro.company_id.id)
+            ])
+            ro.picking_type_visible = picking_count > 1
```

Comment:
```suggestion
        repair_type_by_company = dict(self.env['stock.picking.type']._read_group([
                ('code', '=', 'repair_operation'),
                ('company_id', 'in', self.company_id.ids)
            ], groupby=['company_id'], aggregates=['__count']))
        for ro in self:
            ro.picking_type_visible = repair_type_by_company.get(ro.company_id, 0) > 1
```
Might be better to use `_read_group` instead.

Side note just to make sure, we only want to display the picking type once there is 2+ repair type for that company?